modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -1521,5 +1660,45 @@ operator<<(std::ostream & out, const FeatureFloodCount::FeatureData & feature)
   return out;
 }
 
+/*****************************************************************************************
+ ******************************* Utility Routines ****************************************
+ *****************************************************************************************
+ */
+void
+updateBBoxExtremesHelper(MeshTools::BoundingBox & bbox, const Point & node)
+{
+  for (unsigned int i = 0; i < LIBMESH_DIM; ++i)
+  {
+    bbox.min()(i) = std::min(bbox.min()(i), node(i));
+    bbox.max()(i) = std::max(bbox.max()(i), node(i));
+  }
+}
+
+void
+updateBBoxExtremesHelper(MeshTools::BoundingBox & bbox, const Elem & elem)
+{
+  for (auto node_n = decltype(elem.n_nodes())(0); node_n < elem.n_nodes(); ++node_n)
+    updateBBoxExtremesHelper(bbox, *(elem.get_node(node_n)));
+}
+
+bool
+areElemListsMergeable(std::list<dof_id_type> elem_list1,
```

Comment:
Do you really want to pass these lists to this function by value? Seems like they could be (const?) references...